src/binary-search-tree.js

```diff
@@ -5,6 +5,49 @@ class BinarySearchTree {
     this.left = null;
     this.right = null;
   }
+  insert(value) {
+    const node = new BinarySearchTree(value);
+    function recurse(bst) {
```

Comment:
Your recursive function never returns, so it will end up blowing the stack when it gets executed. Remember to construct your base case correctly in a way that you know the base case will eventually be reached by your recursive function, and that once that case has been reached, that your recursive function returns, otherwise it won't stop running.